packages/java/endpoint/src/main/java/com/vaadin/hilla/route/RouteUtilConfiguration.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2000-2025 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.hilla.route;
+
+import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+
+import com.vaadin.flow.internal.hilla.FileRouterRequestUtil;
+
+@AutoConfiguration
+class RouteUtilConfiguration {
+
+    @Bean
+    @ConditionalOnMissingBean(FileRouterRequestUtil.class)
```

Comment:
`FileRouterRequestUtil` is an internal API, introduced to create a bridge between Flow and Hilla to configure Spring Security. Could we remove the `@ConditionalOnMissingBean`?
I guess we don't want users to override it implementing custom behaviors.